datasets/uy/pep/crawler.py

```diff
@@ -0,0 +1,55 @@
+from zavod import Context
+from typing import Dict, List, Optional
+from pantomime.types import XLSX
+import openpyxl
+from normality import slugify
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+
+def sheet_to_dicts(sheet):
+    headers: Optional[List[str]] = None
+    for row in sheet.rows:
+        cells = [c.value for c in row]
+
+        if headers is None and all(cells):
+            headers = [slugify(h) for h in cells]
+            continue
+
+        if headers:
+            row = dict(zip(headers, cells))
+            yield row
+
+
+def parse_sheet_row(context: Context, row: Dict[str, str]):
+    person = context.make("Person")
+
+    id_number = row.pop("c-i")
+    person_name = row.pop("nombre")
+    role = row.pop("cargo")
+
+    person.id = context.make_slug(id_number, prefix="ur-cedula")
+    person.add("idNumber", id_number)
+    person.add("name", person_name)
+
+    organization_name = row.pop("organismo")
+
+    organization = context.make("Organization")
+    organization.id = context.make_slug(organization_name, prefix="ur-cedula")
+    organization.add("name", organization_name)
+
+    position = h.make_position(
+        context, role, country="uy", organization=organization, lang="spa"
+    )
+
+    context.emit(person, target=True)
+    context.emit(position)
```

Comment:
Did you notice in targets.nested.json that the positions aren't there? Don't worry, the third milestone where you add Occupancy entites will link people to their positions, which will result in the positions being populated nested in the person objects.